front/lib/resources/space_resource.ts

```diff
@@ -182,9 +182,11 @@ export class SpaceResource extends BaseResource<SpaceModel> {
 
   static async listWorkspaceSpaces(
     auth: Authenticator,
-    options?: { includeConversationsSpace?: boolean }
+    options?: { includeConversationsSpace?: boolean; includeDeleted?: boolean }
```

Comment:
I'm not sure to understand why we need this. We have `softDeleteSpaceAndLaunchScrubWorkflow` so if a space is soft deleted then it must have an associated workflow to scrub and hard delete. Do you have occurrences where it's not the case?

Reply:
Yes I have one in production

Reply:
Can you share it, this should not happen. I don't think we should handle this case in this workflow deletion code 🤔. I'd rather fix the sole (few?) occurrences we have in production.

Reply:
Well we might as well have a resilient workspace deletion workflow as we can't guarantee that the scrub workflow is not cancelled by a human because failing for a reason or another.

Reply:
Moving forward with this to unblock the workflow as it does not change any behavior

Reply:
(Workspace ID in prod is 49638)